platform/android/plugin/godot_plugin_config.h

```diff
@@ -86,17 +86,18 @@ struct PluginConfig {
 
 /*
  * Set of prebuilt plugins.
+ * Currently unused, this is just for future reference:
```

Comment:
I guess we could completely remove the prebuilt plugin logic as I guess we won't need it in the future.